src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -140,7 +140,23 @@ def _bigquery(self) -> bigquery.Client:
             return bigquery.Client(project=self._source.project)
 
     def _run_sql(self, query: str) -> BigQueryRows:
-        return self._bigquery.query(query)
+        backoff_intervals = (10, 20, 40, 80)
+        assert (sum(backoff_intervals) < config.contribution_lambda_timeout)
```

Comment:
Whoops, I thought I had already mentioned this. Redundant parentheses:
```suggestion
        assert sum(backoff_intervals) < config.contribution_lambda_timeout
```